crates/gen/src/types/delegate.rs

```diff
@@ -150,7 +150,7 @@ impl Delegate {
                     invoke: #impl_name::invoke,
                     #phantoms
                 };
-                pub fn new(invoke: F) -> #name {
+                pub fn make(invoke: F) -> #name {
```

Comment:
What's the reason for the name change?

Reply:
Calling it new and *not* returning `Self` results in a clippy warning.